fast-af/fast-checkout-woocommerce
# PINT-387 Add Fast button widgets

# Description

Add WordPress widgets to display the Fast Checkout buttons and the Fast Login button.

# Testing instructions

1. Login to the [WordPress admin on the staging site](https://fasttestdev.wpcomstaging.com/wp-admin/).
2. Go to the [Widgets manager](https://fasttestdev.wpcomstaging.com/wp-admin/widgets.php).
3. Add the Fast Cart Button, Fast Product Button, and Fast Login Button widgets to the "Footer" position.
4. Verify that the widgets are displayed in the footer when appropriate.

# Checks

@brikr @ilkerulutas this is ready for review.
